@app/db/.gmrc

```diff
@@ -68,7 +68,7 @@
     "!afterReset.sql",
     {
       "_": "command",
-      "command": "DATABASE_URL=\"${GM_DBURL}\" yarn workspace @app/worker install-db-schema"
+      "command": "cross-env DATABASE_URL=$GM_DBURL yarn workspace @app/worker install-db-schema"
```

Comment:
In bash, `$GM_DBURL` should be quoted to ensure that it doesn't get expanded to multiple arguments/etc. Is this problematic for cross-env?

Reply:
Yes it does not work otherwise. I am not sure if it can lead to unwanted behavior?

Reply:
It's an issue if the envvar contains special characters; e.g. if it contains spaces (e.g. in the database password) then it will break the command. This is solveable by being careful to URL encode the GM_DBURL but it's risky to rely on this.

The [cross-env README shows this working with quotes](https://www.npmjs.com/package/cross-env#usage)*; I think it might have been the braces causing issues, try it with just quotes:

```suggestion
      "command": "cross-env DATABASE_URL=\"$GM_DBURL\" yarn workspace @app/worker install-db-schema"
```

\* well; it shows `$` placeholders working within string args, and it shows quoted envvar values, I'm hoping that these two techniques can be combined.